stand/efi/loader/bootinfo.c

```diff
@@ -339,7 +339,12 @@ bi_load(char *args, vm_offset_t *modulep, vm_offset_t *kernendp, bool exit_bs)
 	vm_offset_t size;
 	char *rootdevname;
 	int howto;
-	bool is64 = sizeof(long) == 8;
+	bool is64 =
+#ifdef __i386__
+	true;
+#else
+	sizeof(long) == 8;
+#endif
```

Comment:
Isn't the above always true now?

Reply:
Yes, that's the point, to force the alignment that the 64-bit kernel would expect.

Reply:
Oh, I think I missed your point here. You meant its always true as in we are always loading a 64-bit kernel?

I thought 32-bit arm was still supported, if not then yea we can just remove the checks.